README.md

```diff
@@ -1,320 +1,353 @@
 # Abbreviations in code
 
-The list of the abbreviations you can find and use in software environments.
-
-## Index
-
--  [Goal](#goal)
--  [Guide](#guide)
--  [List](#list)
--  [Contributing](./CONTRIBUTING.md)
-
-## A good abbreviation is:
-
--  `global` • understood by everyone
--  `significant` • seen for the first time you still know what it means
--  `absolute` • not related to the context
--  `short` • taking out one letter is not an abbreviation
-
-## Goal
-
-Giving good names to variables, functions, classes, files etc. can be difficult. Long names are always clear, but take time to read, take up space and are hard to remember. Choosing a shorter form is better.
-
+Giving good names to variables, functions, files etc. can be difficult. Long names are always clear, but take time to read, space and are hard to remember.
+Choosing a shorter form is better.
 If you use the abbreviations below, your naming will become more coherent, logical and understandable to other programmers (even newbies).
 
+## [CONTRIBUTING](./.github/CONTRIBUTING.md)
+
 ## Guide
 
-Abbreviations are in key&ndash;value format. Multiple choices are sorted from most to least used. Colored circles signify the degree of recommendation:
+Colored circles before the abbr mean the degree of recommendation:
 
-- 🟢 Recommended
-- 🟡 Context-sensitive
-- 🔴 Not recommended
+-  🟢 Recommended
+-  🟡 Context-sensitive
+-  🔴 Not recommended
 
-Unrecommended ones can still be considered. The most important thing is to remain consistent in your code.
+Unrecommended abbrs can still be considered.
+The most important thing is to remain consistent in your code.
+Context abbrs are more suitable in certain cases. The context is specified between { } after the abbr.
```

Comment:
Can we update :yellow_circle:  as
-  🟡 Context-sensitive (You can see the context in `{ }` after the abbreviation)

Reply:
yes